mobile/src/components/Button.tsx

```diff
@@ -27,8 +27,8 @@ export default function Button({
   ...props
 }: Props) {
   //warning: do not touch the extra spaces at the beginning or it will break the styling
-  let baseStyle = "h-7 items-center justify-center";
-  let textStyle = "text-white margin mx-5 font-semibold";
+  let baseStyle = "h-10 items-center justify-center";
+  let textStyle = "text-white margin mx-5 text-xl font-semibold";
```

Comment:
Usually, you don't need a button to be larger than 1rem which is the default size. Is there any reason we are setting text-xl here?

Reply:
The button text was pretty small so I wanted to make it bigger for accessibility.

Reply:
Alright that's reasonable, I'm not sure how nativewind converts this to native styling behind the scenes. Maybe It isn't the same size as on web with the same values